+++
title = "Interacting networks of resistance, virulence and core machinerygenes identified by genome-wide epistasis analysis"
date = 2017-02-01
authors = ["Marcin J Skwark", "Nicholas J Croucher", "Santeri Puranen", "Claire Chewapreecha", "Maiju Pesonen", "Ying Ying Xu", "Paul Turner", "Simon R Harris", "Stephen B Beres", "James M Musser", "Julian Parkhill", "Stephen D Bentley", "Erik Aurell", "Jukka Corander"]
publication_types = ["2"]
abstract = "Recent advances in the scale and diversity of population genomicdatasets for bacteria now provide the potential for genome-widepatterns of co-evolution to be studied at the resolution ofindividual bases. Here we describe a new statistical method,genomeDCA, which uses recent advances in computationalstructural biology to identify the polymorphic loci under thestrongest co-evolutionary pressures. We apply genomeDCA to twolarge population data sets representing the major humanpathogens Streptococcus pneumoniae (pneumococcus) andStreptococcus pyogenes (group A Streptococcus). For pneumococcuswe identified 5,199 putative epistatic interactions between1,936 sites. Over three-quarters of the links were between siteswithin the pbp2x, pbp1a and pbp2b genes, the sequences of whichare critical in determining non-susceptibility to beta-lactamantibiotics. A network-based analysis found these genes werealso coupled to that encoding dihydrofolate reductase, changesto which underlie trimethoprim resistance. Distinct from theseantibiotic resistance genes, a large network component of 384protein coding sequences encompassed many genes critical inbasic cellular functions, while another distinct componentincluded genes associated with virulence. The group AStreptococcus (GAS) data set population represents a clonalpopulation with relatively little genetic variation and a highlevel of linkage disequilibrium across the genome. Despite this,we were able to pinpoint two RNA pseudouridine synthases, whichwere each strongly linked to a separate set of loci across thechromosome, representing biologically plausible targets ofco-selection. The population genomic analysis method appliedhere identifies statistically significantly co-evolving locuspairs, potentially arising from fitness selectioninterdependence reflecting underlying protein-proteininteractions, or genes whose product activities contribute tothe same phenotype. This discovery approach greatly enhances thefuture potential of epistasis analysis for systems biology, andcan complement genome-wide association studies as a means offormulating hypotheses for targeted experimental work."
featured = false
publication = "*PLoS Genet.*"
+++

